compiler/rustc_codegen_ssa/src/mir/operand.rs

```diff
@@ -419,7 +418,14 @@ impl<'a, 'tcx, V: CodegenObject> OperandValue<V> {
                     bx.store_with_flags(val, dest.llval, dest.align, flags);
                     return;
                 }
-                base::memcpy_ty(bx, dest.llval, dest.align, r, source_align, dest.layout, flags)
+                bx.memcpy_known_size(
```

Comment:
🤔 why isn't this also just using `typed_place_copy`?

also, `memcpy_known_size` has one callsite (this one) -- you should inline it. i don't really see the value of exposing it if it's only being used once, seems to add more confusion to the api imo.